src/async.js

```diff
@@ -0,0 +1,16 @@
+(() => {
```

Comment:
No need for an IIFE on the back end since you don't have to worry about scope so much - you can either just `module.exports` the whole function, or create the function as a variable and export it (as you're already doing - maybe switch it to ES6 syntax for consistency however?)